src/index.ts

```diff
@@ -51,21 +57,26 @@ const secp256k1 = {
     return { privateKey, publicKey }
   },
 
-  sign(message, privateKey) {
+  sign(message, privateKey): string {
```

Comment:
In general, I'm to the point of wanting to pass byte arrays around (and I *think* this is the general consensus on the team). 

hex is really an encoding layer on bytes so it's more error prone (i.e. I can make a string and it's not valid hex bytes, but I cannot make an array of bytes that  isn't valid hex). I realize you're just passing through data here so NBD but if you had a  hex->bytes encoder around that you can easily add that would be super cool.

Reply:
I don't see sufficient benefit to making this a breaking change, but in a separate PR, we can certainly add a new method that takes bytes and returns bytes.